trtools/annotaTR/tests/test_annotaTR.py

```diff
@@ -85,6 +88,25 @@ def test_OutTypes(args, vcfdir):
     args.outtype = ["pgen"]
     retcode = main(args)
     assert retcode==0
+    args.outtype = ["gzvcf"]
+    retcode = main(args)
+    assert retcode==0
+    args.outtype = ["gzvcf", "pgen"]
+    retcode = main(args)
+    assert retcode==0
+    args.outtype = ["gzvcf", "vcf"]
+    retcode = main(args)
+    assert retcode==1
+    # Should get pgen error if input VCF has fewer
+    # TRs than ref panel
+    args.vcf = os.path.join(vcfdir, "beagle", "beagle_imputed_noTRs.vcf.gz")
+    args.ref_panel = os.path.join(vcfdir, "beagle", "beagle_tinyrefpanel.vcf.gz")
+    args.match_refpanel_on = "locid"
+    args.dosages = "bestguess_norm"
+    args.outtype = ["pgen"]
+    args.out = "test"
+    retcode = main(args)
+    assert retcode==1
```

Comment:
I think there might be some leftover files from running this test
```suggestion
    assert retcode==1
    for fname_ext in ("pgen", "pvar", "psam"):
        if os.path.exists("test."+fname_ext):
            os.remove("test."+fname_ext)
```